proiect/src/shared/Rating.java

```diff
@@ -0,0 +1,66 @@
+package shared;
+
+public class Rating {
+	
+	private double score;
+	
+	private int numberOfRatings;
+	
+	public Rating() {
```

Comment:
it can be empty constructor, defaults will be 0 already

Reply:
Resolved in issue #3